salt/modules/zypperpkg.py

```diff
@@ -439,29 +433,27 @@ def __call__(self, pkg_name, pkg_version):
                     for vrs in self._get_scope_versions(self._get_available_versions())
                 ]
             )
-            return versions and "{0}{1}".format(self._op or "", versions[-1]) or None
+            return versions and "{}{}".format(self._op or "", versions[-1]) or None
 
     def _get_available_versions(self):
         """
         Get available versions of the package.
         :return:
         """
         solvables = self.zypper.nolock.xml.call(
-            "se", "-xv", self.name
+            "se", "-v", self.name, success_retcodes=NOT_FOUND_RETCODES
```

Comment:
@aplanas why do you remove -x flag?

Reply:
`--xmlout, -x            Switch to XML output.` is implicit in the `.xml` part of the chain, that add the parameter here:

https://github.com/saltstack/salt/blob/33733c8cae2fe03ec010c787d90e95223fbd2b8d/salt/modules/zypperpkg.py#L300